src/components/SinListItem.jsx

```diff
@@ -1,13 +1,25 @@
 import { TrashIcon } from "@heroicons/react/16/solid";
+import { useSortable } from "@dnd-kit/react/sortable";
+
+const SinListItem = ({ sinItem, onRemoveSinItem, index }) => {
+  const { ref } = useSortable({
+    id: sinItem.id ?? sinItem.text + Math.random(),
```

Comment:
I think `Math.random` here is going to generate a different ID on each render, which seems like a problem (assuming dnd-kit wants a stable identifier). The ID can be a string, so we can just use the text itself (but that would be a problem if the user adds multiple custom sins). Perhaps we can capture a timestamp when saving a custom sin and use that as a stable identifier (tracked when creating the sin in the ConfessIt component)?